### PR TITLE
Do not allow duplicates invites

### PR DESCRIPTION
**Pull request by norkans7**
Fix https://github.com/rapidpro/rapidpro/issues/1438

### CODE REVIEW

File: temba/orgs/views.py [hunk 1]
```diff
@@ -1653,7 +1653,7 @@ def clean_invite_emails(self):
                 emails = self.cleaned_data["invite_emails"].lower().strip()
                 existing_users_emails = set(
                     list(self.org.get_users().values_list("username", flat=True))
-                    + list(self.org.invitations.values_list("email"))
+                    + list(self.org.invitations.filter(is_active=True).values_list("email", flat=True))
```
Comment (rowanseymour) [inline temba/orgs/views.py:1656]: seems like this will still allow duplicates in the list of new emails, no?

**Review by rowanseymour — CHANGES_REQUESTED** [review]
see comment

File: temba/orgs/views.py (hunk 1 above)
Comment (norkans7) [reply temba/orgs/views.py:1656]: the bug was the missing `flat=True`

Comment (rowanseymour) [reply temba/orgs/views.py:1656]: Yeah that fixes not checking existing invitations but I think you also need to check there aren't duplicates in the invitations being submitted

Comment (norkans7) [reply temba/orgs/views.py:1656]: Updated

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**